src/__tests__/internal/deployment.test.js

```diff
@@ -448,10 +655,6 @@ describe('Deployment', () => {
       await deployment.cancel()
 
       expect(core.error).toHaveBeenCalledWith(`Canceling Pages deployment failed`, expect.anything())
-
-      artifactExchangeScope.done()
-      createDeploymentScope.done()
-      cancelDeploymentScope.done()
```

Comment:
🛑 **Blocker**

As ugly as this pattern is, including the `.done()` calls is fundamentally different from omitting them, as they will [intentionally throw errors](https://github.com/nock/nock#expectations) if their mocked endpoints were never called.

I'd like to see these reinstated, as well as added to your new tests. If those are resulting in errors thrown, then either (a) something is wrong, or (b) we are mocking more endpoints than necessary for a given test.

Reply:
Got it!